lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -288,7 +288,14 @@ SmallVector<unsigned> getOrder(Attribute layout) {
     auto distributedLayout = cast<DistributedEncodingTrait>(layout);
     auto rank = distributedLayout.getWarpsPerCTA().size();
     SmallVector<unsigned> order(rank);
-    std::iota(order.rbegin(), order.rend(), 0);
+    auto nvidiaMma = dyn_cast<NvidiaMmaEncodingAttr>(layout);
+    if (nvidiaMma && nvidiaMma.isHopper()) {
+      // Hopper WGMMA requires that the 4 warps in a warp group be laid out along
+      // the M-dimension; so here we rasterize M-first
+      std::iota(order.begin(), order.end(), 0);
```

Comment:
This is the order of WGMMA resultant but not its dot operand. I don't see why the order is [0, 1, 2, 3, ...]

Reply:
Maybe the method you wanted to use is `getWarpOrder`?

Reply:
I see, that makes sense. I'll modify `SharedToDotMMAv2OrV3` to call `getWarpOrder` instead of `getOrder(MMALayout)`